integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/MessageOutController.java

```diff
@@ -179,14 +180,39 @@ public ResponseEntity createResource(
     }
 
     private void setDefaults(ConversationResource cr) {
-        cr.setSenderId(isNullOrEmpty(cr.getSenderId()) ? props.getOrg().getNumber() : cr.getSenderId());
-        InfoRecord senderInfo = sr.getInfoRecord(cr.getSenderId());
-        InfoRecord receiverInfo = sr.getInfoRecord(cr.getReceiverId());
+        InfoRecord senderInfo = sr.getInfoRecord(props.getOrg().getNumber());
+        if (cr.getSender() == null) {
+            cr.setSender(Sender.of(senderInfo.getIdentifier(), senderInfo.getOrganizationName()));
+        } else {
+            if (isNullOrEmpty(cr.getSender().getSenderId())) {
+                cr.getSender().setSenderId(senderInfo.getIdentifier());
+            }
+            if (isNullOrEmpty(cr.getSender().getSenderName())) {
+                cr.getSender().setSenderName(senderInfo.getOrganizationName());
+            }
+        }
+
+        // FIXME: temp fix, see MOVE-695
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1134)